src/sockets/commands/admin/areset.ts

```diff
@@ -0,0 +1,42 @@
+import { Command } from '../types';
+import { SocketUser } from '../../types';
+import User from '../../../models/user';
+import seasonNumber from '../../../util/seasonNumber';
+
+export const areset: Command = {
+  command: 'reset',
+  help: "/reset all plyers's rank data and start a new season",
+  run: async (args: string[], socket: SocketUser) => {
+    try {
+      const users = await User.find({});
+      for (const user of users) {
+        const previousRankData = {
+          seasonNumber: seasonNumber.getSeasonNumber() ,
+          rating: user.playerRating,
+          ratingBracket: user.ratingBracket,
+        };
+        user.previousRankData.push(previousRankData);
+        user.playerRating = 1500;
+        user.ratingBracket = 'silver';
+        user.rd = 350;
+        user.volatility = 0.06;
```

Comment:
Let's have these default constants in a separate file so they're not duplicated across the model and in here.

Reply:
Sure, I will think how to do that.